pypsa-to-iamc

```diff
@@ -0,0 +1,634 @@
+            
+            var[sh]['Installed capacity|Electricity|Gases|Hydrogen'] = 0.001*((n.links.efficiency.filter(like ='H2 Fuel Cell').filter(like =country)
+                 *n.links.p_nom_opt.filter(like ='H2 Fuel Cell').filter(like =country)).sum())
+
+            #Capacity : Oil 
+
+            var[sh]['Installed capacity|Electricity|Liquids|Fossil'] = 0.001*((n.links.efficiency.filter(like ='oil-').filter(like =country)
+                 *n.links.p_nom_opt.filter(like ='oil-').filter(like =country)).sum())
+
+            #Capacity : Power to gas 
+            
+            var[sh]['Installed capacity|P2G|Electricity'] = 0.001*((n.links.efficiency.filter(like ='Sabatier').filter(like =country)
+                 *n.links.p_nom_opt.filter(like ='Sabatier').filter(like =country)).sum()+ 
+                 (n.links.efficiency.filter(like ='helmeth').filter(like =country)*n.links.p_nom_opt.filter(like ='helmeth').filter(like =country)).sum()+ 
+                 (n.links.efficiency.filter(like ='H2 Electrolysis').filter(like =country)*n.links.p_nom_opt.filter(like ='H2 Electrolysis').filter(like =country)).sum()+
+                 (n.links.efficiency.filter(like ='Fischer-Tropsch').filter(like =country)*n.links.p_nom_opt.filter(like ='Fischer-Tropsch').filter(like =country)).sum()) 
+            
+            #Capacity : hydro (reservoir, ror)
+
+            var[sh]['Installed capacity|Electricity|Hydro|river'] = 0.001*n.generators.p_nom_opt.filter(like ='ror').filter(like =country).sum()
+            var[sh]['Installed capacity|Electricity|Hydro|dam'] = 0.001*n.storage_units.p_nom_opt.filter(like ='hydro').filter(like =country).sum()
+            var[sh]['Installed capacity|Electricity|Hydro'] = var[sh]['Installed capacity|Electricity|Hydro|river']+var[sh]['Installed capacity|Electricity|Hydro|dam']
+           
+            
+            #Capacity : storage (PHS, battery, H2 storage)
+            
+            var[sh]['Installed capacity|Flexibility|Electricity Storage|Medium duration'] = 0.001*((n.storage_units.p_nom_opt.filter(like ='PHS').filter(like =country).sum()
+             +n.stores.e_nom_opt.filter(like ='H2').filter(like =country)).sum())
+            
+            var[sh]['Installed capacity|Flexibility|Electricity Storage|Short duration'] = 0.001*((n.stores.e_nom_opt.filter(like ='battery').filter(like =country)
+             +n.stores.e_nom_opt.filter(like ='Li ion').filter(like =country)).sum())
+            
+            var[sh]['Installed capacity|Flexibility|Electricity Storage'] =  var[sh]['Installed capacity|Flexibility|Electricity Storage|Medium duration']+\
+            var[sh]['Installed capacity|Flexibility|Electricity Storage|Short duration']
+            
+            #Capacity : Interconnect 
+            
+            var[sh]['Installed capacity|Flexibility|Interconnect Importing Capacity'] = 0.001*((n.lines.s_nom_opt[[i for i in n.lines.index if country in n.lines.bus0[i]or country in n.lines.bus1[i]]]).sum()
```

Comment:
I checked and it works for separate countries (because we have one DE0-AT0 and it's calculated once for Germany and once for Austria separately). The problem is that when these are all added to the 'EU' sheet, the number is twice the actual capacity. I'm not sure how to fix this in the code without adding an exception to the final part. We could also just fix the number in the final Excel for the 'EU' sheet.
side note: for DC links, I excluded inner-country links, is that right?